canonical/ubuntu.com
# [copy update] WD-3059 update `/landscape/install`

## Done

- Update page based on requested copy docs update

## QA

- Check out the [demo here](url) and copy with [copy doc](https://docs.google.com/document/d/1tU1z9PUm_kSwGJEsopJibE4gI8Gfv_3Dw_qN2_io_ao/edit#).
  - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

- [WD-3059](https://warthogs.atlassian.net/browse/WD-3059)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
